### DISCUSSION TITLE
想问下BE支持绑核吗？

### DISCUSSION THREAD

**Discussion by Hoxxx** [apache/doris#20549] [A - General / Q&A] [answered]
doris：1.2.4.1/2.0.0
主机128c，4个node，硬件工程师说建议软件全部做绑核；如果支持的话，是否可以在一台主机上运行多个BE？（比如运行三个BE，每个BE绑一个node）

**Comment by yagagagaga** [comment, marked as answer]
Usually, it is recommended that BE monopolize one machine, and you can bind the CPU core through `taskset` or `cpuset`. In 2.0, Doris introduced the [Workload Group](https://doris.apache.org/zh-CN/docs/dev/admin-manual/workload-group).

**Reply by Hoxxx** [reply]
OK，不过还是希望有官方绑核的操作，最好可以在配置文件里设置，直接指定node或cpu_number
